src/main/java/Menu.java

```diff
@@ -0,0 +1,73 @@
+import com.eclipsesource.json.Json;
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Menu {
+    private ArrayList<Dish> dishes;
+    private final String JSON_path = "src/jsons/menu.json";
```

Comment:
немного неправильный кодстайл, нужно
```suggestion
    private final String JSON_PATH = "src/jsons/menu.json";
```